bin/add_test_types.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+new_test_types = [
```

Comment:
It would be great if the data elements could be prompts so that this can be used in the future. But as it is it will work.

Reply:
> It would be great if the data elements could be prompts so that this can be used in the future. But as it is it will work.

This is noted